LD.pm

```diff
@@ -161,16 +165,30 @@ sub new {
 
   $pop_name ||= '1000GENOMES:phase_3:CEU';
 
+  my @pop_names = ();
+
+  if ($pop_name =~ /^populations=/) {
+    $pop_name =~ s/populations=//;  
+    push @pop_names, split('&', $pop_name);
+  } else {
+    push @pop_names, $pop_name;
+  }
+
+
   $r2_cutoff = 0.8 unless defined $r2_cutoff;
 
   my $pop_adap = $reg->get_adaptor('human', 'variation', 'population')
     || die "Failed to get population adaptor\n";
 
   my $valid_pops = $pop_adap->fetch_all_LD_Populations();
-  my ($pop) = grep {$_->name eq $pop_name} @$valid_pops;
-  die "Invalid population '$pop_name'; valid populations are:\n".join(", ", map {$_->name} @$valid_pops)."\n" unless $pop;
+  my @populations = ();
+  foreach my $pop_name (@pop_names) {
+    my ($pop) = grep {$_->name eq $pop_name} @$valid_pops;
+    die "Invalid population '$pop_name'; valid populations are:\n".join(", ", map {$_->name} @$valid_pops)."\n" unless $pop;
+    push @populations, $pop;
```

Comment:
Should it rather be a hash instead of an array, just in case we might have duplicated populations ?
e.g.:
```
my %populations;
foreach my $p (@$pop) {
  $populations->{$p} = 1;
}
...
$self->{populations} = [sort(keys(%populations))];
```